src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const besoreWeekDiscount = 20;
+  const afterWeekDiscount = 50;
+  let discount = 0;
+
+  if (days >= 3) {
```

Comment:
<img width="806" alt="transOnVacChlist" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/77551820/17f291a1-e5f6-48b5-a951-285b4c138053">

pls move all numbers to consts

Reply:
> <img alt="transOnVacChlist" width="806" src="https://private-user-images.githubusercontent.com/77551820/305918636-17f291a1-e5f6-48b5-a951-285b4c138053.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgzMzkyMDgsIm5iZiI6MTcwODMzODkwOCwicGF0aCI6Ii83NzU1MTgyMC8zMDU5MTg2MzYtMTdmMjkxYTEtZTVmNi00OGI1LWE5NTEtMjg1YjRjMTM4MDUzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE5VDEwMzUwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ4NzQxOWEyMGZlN2FiMWUwYmRlNTg0NzBjM2MxMWM4N2QwMGY4MGVlNDVkOGJlY2E2MWU3YmFkMjY1OWNkZTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zdJ7_CIv9WE25TMdOPWrXzQRz9-ya-cJ4y2FiuzekZI">
> pls move all numbers to consts

thx now i get it